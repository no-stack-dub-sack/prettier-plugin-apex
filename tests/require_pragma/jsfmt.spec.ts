runSpec(__dirname, ["apex"], { requirePragma: true });
