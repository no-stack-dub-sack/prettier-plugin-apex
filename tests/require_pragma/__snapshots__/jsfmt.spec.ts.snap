// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Format apex: ClassWithFormatPragma.cls: ClassWithFormatPragma.cls 1`] = `
/**
 * @format
 */
class ClassWithFormatPragma {}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/**
 * @format
 */
class ClassWithFormatPragma {
}

`;

exports[`Format apex: ClassWithPrettierPragma.cls: ClassWithPrettierPragma.cls 1`] = `
/**
 * @prettier
 */
class ClassWithPrettierPragma {}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/**
 * @prettier
 */
class ClassWithPrettierPragma {
}

`;

exports[`Format apex: ClassWithShortFormatPragma.cls: ClassWithShortFormatPragma.cls 1`] = `
/** @format */
class ClassWithShortFormatPragma {}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/** @format */
class ClassWithShortFormatPragma {
}

`;

exports[`Format apex: ClassWithShortPrettierPragma.cls: ClassWithShortPrettierPragma.cls 1`] = `
/** @prettier */
class ClassWithShortPrettierPragma {}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/** @prettier */
class ClassWithShortPrettierPragma {
}

`;

exports[`Format apex: ClassWithoutPragma.cls: ClassWithoutPragma.cls 1`] = `
class ClassWithoutPragma {}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
class ClassWithoutPragma {}

`;
